about.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap" rel="stylesheet">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/e8963cdebe.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+    <title>Nairobi Classical Festival</title>
+</head>
+<body>
+    <main>
+        <header>
+            <div class="desktop-only top-header">
+                <i class="fa fa-facebook" aria-hidden="true"></i>
+                <i class="fa fa-twitter" aria-hidden="true"></i>
+                <p>Translate</p>
+                <p>Logout</p>
+            </div>
+    
+            <div class="desktop-only bottom-header">
+                <div class="project-logo">
+                    <a href="index.html"><img src="images/main-logo.svg" alt="logo-image"></a>
+                </div>
+                <div class="desktop-header-content">
```

Comment:
- [ ] [optional] As mentioned by the previous reviewer kindly use the section tag in place of the div tag.